lib/esc_pos_utils/src/font_config/find_multuples_use_case.dart

```diff
@@ -0,0 +1,27 @@
+class FindMultiplesUseCase {
```

Comment:
Typo in file name
Also shouldn't the name be something related to dividend or divisor?